### PR TITLE
Provide IRelationBrokenEvent to be able to distinguish the event

### PR DESCRIPTION
**Pull request by vangheem**

### CODE REVIEW

**Comment by icemac** [comment]
@vangheem I fixed the Travis tests on master, could you please rebase your PR?

**Comment by icemac** [comment]
Huh? Was closing this PR intensional?

File: CHANGES.rst
Comment (mgedmin) [inline CHANGES.rst]: You've accidentally indented this line, which will likely break ReStructuredText formatting.

File: src/z3c/relationfield/interfaces.py
Comment (mgedmin) [inline src/z3c/relationfield/interfaces.py:108]: (A docstring would be nice.)

**Review by mgedmin — COMMENTED** [review]
LGTM, if you fix the ReST mistake.

**Review by mgedmin — APPROVED** [review]

**Comment by icemac** [comment]
I think this PR should be merged after #12 as that branch was used for a release. It should be merged to master before anything else.

**Comment by icemac** [comment]
@vangheem After I merged #12 now, could you please rebase your PR?

**Comment by icemac** [comment]
@vangheem Is this PR still valid?

**Comment by vangheem** [comment]
Closing this for https://github.com/zopefoundation/z3c.relationfield/pull/18

**Pull request closed without merging**
